account.go

```diff
@@ -8,4 +8,5 @@ type Account struct {
 	SubDomain  string `json:"subdomain"`
 	Username   string `json:"username"`
 	Password   string `json:"password"`
+	Server     string `json:"server"`
```

Comment:
Sorry to bikeshed but what do you think about picking a name that's a little bit more reflective of the fact that it's a URL? 

One other thought: I think it would be useful to include a comment here that the value might be empty for legacy accounts.

```suggestion
	// ServerURL contains the URL of the acme-dns server the Account was registered with 
	// (may be empty for  Account instances registered before this field was added).
	ServerURL     string `json:"server_url"`
```